installing/installing_vsphere/installing-vsphere-agent-based-installer.adoc

```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-vsphere-agent-based-installer"]
+= Installing a cluster on vSphere using the Agent-based installer
+include::_attributes/common-attributes.adoc[]
+:context: installing-vsphere-agent-based-installer
+
+toc::[]
+
+The Agent-based installation method provides the flexibility to boot your on-premises servers in any way that you choose. It combines the ease of use of the Assisted Installation service with the ability to run offline, including in air-gapped environments.
+
+Agent-based installation is a subcommand of the {product-title} installer. It generates a bootable ISO image containing all of the information required to deploy an {product-title} cluster with an available release image.
+
+[role="_additional-resources"]
+[id="additional-resources_installing-vsphere-agent-based-installer"]
+== Additional resources
+
+* xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#preparing-to-install-with-agent-based-installer[Preparing to install with the Agent-based installer]
```

Comment:
This looks good to me. If you want to be extra thorough you can add another link to the actual install procedure, "[Installing a cluster with Agent-based Installer](https://docs.openshift.com/container-platform/4.14/installing/installing_with_agent_based_installer/installing-with-agent-based-installer.html)", but I think just this one link to the prep page is enough as well.

Reply:
This also makes me realize I need to go through the Agent docs and capitalize "Installer"  😅

Reply:
I would prefer to leave this link only. I think that this topic is where the rubber meets the road in terms of introducing the installation method.